lib/viking/model/reflections/belongs_to_reflection.js

```diff
@@ -6,13 +6,11 @@ Viking.Model.BelongsToReflection = Viking.Model.Reflection.extend({
         this.options = _.extend({}, options);
     
         if (!this.options.polymorphic) {
-            this.modelName = this.options.model || {
-                className: name.camelize(),
-                singular:  name,
-                plural:    name.pluralize(),
-                routeKey:   name.split('/').pop().pluralize(),
-                paramKey:   name.split('/').pop()
-            };
+            if (typeof(this.options.modelName) === 'string') {
+                this.modelName = new Viking.Model.Name(this.options.modelName);
```

Comment:
I think in the `belongs_to` and other relationships we should pass the `objectName`, but `Viking.Model.Name` I think should just take one formate not try to support all and in a place like this if `Viking.Model.Name` takes and underscore version the transformation is done here?